tools/util/include/cutlass/util/reference/device/tensor_epilogue.h

```diff
@@ -147,5 +223,9 @@ namespace sycl {
       cutlass::reference::device::detail::TensorPerRowBiasFunc<
           float, cutlass::layout::RowMajor, float,
           cutlass::layout::ColumnMajor>::Params> : std::true_type {};
-  } // namespace sycl
+  template <>
```

Comment:
Referring to Finlay's comment on the Dual GEMM PR, I think this `is_device_copyable` can be removed since we should have a default constructor for the `TensorView` class.
Comment: https://github.com/codeplaysoftware/cutlass-fork/pull/260#discussion_r2000833794
Changes I added for Dual GEMM: https://github.com/codeplaysoftware/cutlass-fork/pull/260/commits/a9a4ba8ff9001e6fdd03efe9556b890de1f3f1de

Reply:
quite right - removed